src/metalnx-web/src/main/resources/views/collections/summary.html

```diff
@@ -12,7 +12,7 @@
 
 			<div class="col-sm-4 col-md-4 col-xs-4 pull-right">
 				<div id="actionmenu" class="pull-right">
-					<div class="btn-group">
+					<div class="btn-group" th:if="${permissionType != 'none' and permissionType != 'read'}">
```

Comment:
consider the opposite logic... to only render if the permissionType is what we want... what if it was set to empty string or something else?  it would render!

```
${permissionType == 'write' or permissionType == 'own'}
```

For ease of reading... does the syntax exist to do something like `permissionType in ('write', 'own')`?

Reply:
That totally makes sense. Making change now.